castai/client.go

```diff
@@ -46,6 +46,7 @@ type Client interface {
 	SendCISCloudScanReport(ctx context.Context, report *CloudScanReport) error
 	PostTelemetry(ctx context.Context, initial bool) (*TelemetryResponse, error)
 	GetSyncState(ctx context.Context, filter *SyncStateFilter) (*SyncStateResponse, error)
+	UpdateImageScanStatus(ctx context.Context, state *ImagesScanRequest) error
```

Comment:
Maybe this name could be more generic like `UpdateImageStatus` to have ability to add more info not only related to scan. Endpoint is already generic /v1/security/insights/images/status